docs/application-services/3-location/2-resolving-location.md

```diff
@@ -0,0 +1,39 @@
+---
+id: resolving
```

Comment:
```suggestion
```

My preference is to avoid using id unless we have a reason to do so. This can make URL resolution confusing as it overrides the filename.

If we remove this id, the filename can be changed to `2-resolving.md` to achieve the same `application-services/location/resolving` url, or leave the filename as is for `application-services/location/resolving-location`.